libs/spark/src/Unstable_SectionMessageTitle/Unstable_SectionMessageTitle.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import makeStyles from '../makeStyles';
+import { StandardProps } from '../utils';
+import Unstable_Typography from '../Unstable_Typography';
+
+export interface Unstable_SectionMessageTitleProps
+  extends StandardProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    Unstable_SectionMessageTitleClassKey
+  > {
+  /**
+   * The content of the component.
+   */
+  title?: string;
+}
+
+export type Unstable_SectionMessageTitleClassKey = 'root';
+
+const useStyles = makeStyles(
+  () => ({
+    root: () => ({ marginBottom: 4 }),
+  }),
+  { name: 'MuiSparkUnstable_SectionMessageTitle' }
+);
+
+const Unstable_SectionMessageTitle = React.forwardRef(
+  function Unstable_SectionMessageTitle(
+    props: Unstable_SectionMessageTitleProps,
+    ref
+  ) {
+    const { title, className, ...other } = props;
+
+    const classes = useStyles();
+
+    return (
+      <span className="pos-3">
+        <Unstable_Typography
+          variant="label"
+          component="div"
```

Comment:
```suggestion
        <div
```
Even though Mui does, I don't think we need to use the whole `Typography` component -- any other variant is far out-of-spec, and neither we nor they make this component inherit the props of the underlying Typography component. 

Just spread `theme.unstable_typography.label` in the styles, and set the color as `theme.unstable_palette.text.heading`.